WcaOnRails/app/views/regulations/scrambles/index.html.erb

```diff
@@ -18,6 +18,17 @@
     <li>Delegates should download TNoodle to run it on a computer. They should not use TNoodle running on a public server (for security reasons).</li>
     <li>Delegates must save all scramble sequences generated for an official competition, and send them with the results (see <%= link_to "Regulation 1c3a", "../#1c3a" %>).</li>
   </ul>
+  <h3 id="scramble-secrecy">Scramble Secrecy</h3>
+    <ul>
+      <li><b>Always</b> encrypt your scrambles in TNoodle with an unguessable password. The password must have nothing to do with the competition or personal data (name, birthdate) of any of the Delegates.</li>
+      <li>If you are displaying scrambles on a digital device, do not give scramblers access to the single long PDF in the folder root! Only give scramblers access to each group's PDF when that group begins. We recommend the following:
+        <ul>
+          <li>Consider using a USB drive to guard and distribute PDFs.</li>
+          <li>Windows: Use <a href="https://www.ghisler.com/">TotalCommander</a> to open PDFs from the encrypted competition ZIP, which prompts for a password every time.</li>
+          <li>macOS or Linux: Use <a href="https://cubing.net/software/files/MollyGuard.dmg">MollyGuard</a> or <code>xtrlock</code> to prevent keyboard or mouse input by the scramblers.</li>
```

Comment:
This is the best technical recommendation the WQAC has found at this point.

I'd prefer to encrypt individual scramble PDFs as an interim solution or spend a weekend building something much better, but I don't want to block Gregor from posting the announcement.

Do you have any alternative suggestions?
(Or time to update TNoodle right now?)

Reply:
I like the suggestion for Windows. Can we provide a similar suggestion for Mac and Linux?

I don't understand how this could involve any TNoodle changes. Could you go into more detail on that, @lgarron?